### PR TITLE
WIP: Adding Hinge, SquaredHinge, CategoricalHinge loss classes

### PR DESCRIPTION
**Pull request by edgarkhanzadyan**
### Summary

Adding Hinge, SquaredHinge, CategoricalHinge loss classes

### PR Overview

- [y] This PR requires new unit tests [y/n] (make sure tests are included)
- [n] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [y] This PR is backwards compatible [y/n]
- [n] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

File: keras/losses.py [hunk 1]
```diff
@@ -430,6 +430,81 @@ def __init__(self,
             from_logits=from_logits)
 
 
+class Hinge(LossFunctionWrapper):
+    """Computes the hinge loss between `y_true` and `y_pred`.
+    `y_true` values are expected to be -1 or 1. If binary (0 or 1) labels are
```
Comment (fchollet) [inline keras/losses.py]: Introduce line breaks (as needed, in all docstrings in this PR)

**Comment by edgarkhanzadyan** [comment]
@fchollet, please, review the PR. I've added the tests and added line breaks in docstrings. Let me know if there is anything to be changed.

**Comment by fchollet** [comment]
Closing outdated PR. Thank you for preparing the PR, though.
